client/src/pages/public/HomePage/HomePage.jsx

```diff
@@ -3,26 +3,74 @@ import Navbar from '../../../components/public/Navbar/Navbar';
 import Hero from '../../../components/public/Hero/Hero';
 import Footer from '../../../components/public/Footer/Footer';
 import Section from '../../../components/public/Section/Section';
+import { Box, Skeleton } from '@mui/material'
+import './HomePage.css'
 
+// const sections = [
```

Comment:
where from this came?

Reply:
Bhaiya, this is just dummy data which we can use during testing phase as actual data is not available currently.